### DISCUSSION TITLE
Formatting the pagination values in a table navigator

### DISCUSSION THREAD

**Discussion by cdecinkoKnight** [quasarframework/quasar#15620] [General - Components / Directives / etc]
When using a q-table and using the standard pagination controls, how can we format the display of numbers? For example, if I have thousands of records, the count shows without commas making the number difficult to read. I would expect to see records 1-10 of 3,567,432 rather than 3567432.

**Comment by metalsadman** [comment]
See qtbale's  api card `pagination-label` https://quasar.dev/vue-components/table#api--qtable

**Reply by cdecinkoKnight** [reply]
I would have to manually create the pagination in order to format that displayed numbers?

**Reply by metalsadman** [reply]
It doesn't say that in the docs so no.

**Reply by cdecinkoKnight** [reply]
pagination-label does not exist on the provided page link.

**Reply by yusufkandemir** [reply]
@cdecinkoKnight 
![image](https://github.com/quasarframework/quasar/assets/6266078/93545a8f-6fff-46d5-9c71-3236549a0b6d)
